src/main/blackbox/blackbox.c

```diff
@@ -1388,8 +1388,8 @@ static bool blackboxWriteSysinfo(void)
         BLACKBOX_PRINT_HEADER_LINE("feedforward_weight", "%d,%d,%d",        currentPidProfile->pid[PID_ROLL].F,
                                                                             currentPidProfile->pid[PID_PITCH].F,
                                                                             currentPidProfile->pid[PID_YAW].F);
-#ifdef USE_INTERPOLATED_SP
-        BLACKBOX_PRINT_HEADER_LINE("ff_interpolate_sp", "%d",               currentPidProfile->ff_interpolate_sp);
+#ifdef USE_FF
```

Comment:
`USE_FEEDFORWARD` will make it possible to actually understand what this means.

Reply:
Since all the internal parameters are now in the form 'ff_abcd' or 'ffAbcd', ad the CLI names are all 'ff_abcd', it's tough to think that a coder wouldn't know that 'ff' was our standard abbreviation for feedforward.
I guess I figured that it would be best to be consistent.
Hence stick with 'ff' as the 'name', wherever possible, rather than sometimes 'ff' and other times 'feedforward'.  
'ff' is quicker and easier to type also :-)